shadow-cljs.edn

```diff
@@ -5,6 +5,8 @@
  ;; "." for /static
  :dev-http {3001 ["static" "."]
             3002 ["public/workspaces" "."]}
+ 
+ :js-options {:js-package-dirs ["node_modules" "tldraw"]}
```

Comment:
This allows loading unpublished npm module

Reply:
Thanks for the sharing!